autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/management/viewing/AutofillManagementListMode.kt

```diff
@@ -178,6 +183,14 @@ class AutofillManagementListMode : DuckDuckGoFragment(R.layout.fragment_autofill
                         credentialsListUpdated(it, state.credentialSearchQuery)
                         parentActivity()?.invalidateOptionsMenu()
                     }
+
+                    if (state.showAutofillEnabledToggle) {
+                        binding.credentialToggleGroup.show()
+                        binding.logins.updateTopMargin(0)
+                    } else {
+                        binding.credentialToggleGroup.gone()
+                        binding.logins.updateTopMargin(16)
```

Comment:
You can use the dimens we have for 0 (keyline_empty) and 16 (keyline_4).

For example, to get the pixel size for 16dp:
`context.resources.getDimensionPixelSize(R.dimen.keyline_4)`